CONDUCT.md

```diff
@@ -0,0 +1,131 @@
+# Conjur Code of Conduct
+
+Conjur exists to make DevOps easier, by making it easy for teams to communicate
```

Comment:
How about "to simplify DevOps" or something so that we don't use easy twice in one sentence?